apps/web/stores/WalkthroughRootStore.ts

```diff
@@ -163,13 +184,15 @@ export class WalkthroughRootStore {
     if (!question) return "";
 
     const answer = this.answerStore.answers[questionId];
-    if (isString(answer)) {
+    if (isString(answer) && "possibleAnswers" in question) {
```

Comment:
Are there other places we are using logic such as this checking for value in string "possibleAnswers" ? Would it make sense to make this a constant that is accessible any file? In the event the json changes or literature changes we have a single place to update?

Reply:
Good call. Refactored!